dotnetthanks-loader.csproj

```diff
@@ -13,7 +13,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="7.0.0" />
```

Comment:
these packages should also be updated from 7 to 8. Let's look at this right after GA.